yorkie/src/main/kotlin/dev/yorkie/document/crdt/CrdtTree.kt

```diff
@@ -542,12 +543,6 @@ internal class CrdtTreeNode(
         get() = removedAt != null
 
     override var value: String = _value.orEmpty()
-        get() {
```

Comment:
I added it because of [this line from the JS SDK](https://github.com/yorkie-team/yorkie-js-sdk/blob/395d052f2a6a3c1a5cef794a2e45b0312a63e945/src/document/crdt/tree.ts#L259), but I guess we can just remove it now, and add a helper function later if it is really necessary.

Reply:
@7hong13 I added it back, but changed CrdtTreeNode to a data class for easier usage (equals, copy, debugging).